modules/checklist/assets/js/app/components/checklist-card-content.js

```diff
@@ -1,6 +1,10 @@
 import { Button, Card, CardActions, CardContent, CardMedia, Link, Typography } from '@elementor/ui';
 import { __ } from '@wordpress/i18n';
 import PropTypes from 'prop-types';
+import { getAndUpdateStep } from '../../utils/functions';
+import { STEP } from '../../utils/consts';
+
+const { IS_MARKED_COMPLETED, IS_ABSOLUTE_COMPLETED, IS_IMMUTABLE_COMPLETED } = STEP;
```

Comment:
Maybe destructure this in one line?
Not crucial.

Reply:
I tried, doesn't work inside an import AFAIK